src/main/java/com/jalasoft/jfc/controller/ImageConverterController.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright (c) 2019 Jalasoft.
+ *
+ * This software is the confidential and proprietary information of Jalasoft.
+ *  ("Confidential Information"). You shall not disclose such Confidential
+ *  Information and shall use it only in accordance with the terms of the
+ *  license agreement you entered into with Jalasoft.
+ */
+
+package com.jalasoft.jfc.controller;
+
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.multipart.MultipartFile;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.io.IOException;
+
+/**
+ *
+ *  Manage ImageConvert Requests.
+ *
+ * @version 0.1.
+ *
+ * @author Enrique Carrizales.
+ */
+@RestController
+@RequestMapping(path = "/convertImage")
+public class ImageConverterController {
+
+    private static final String UPLOADED_FOLDER =
+            "src/main/java/com/jalasoft/jfc/resources/";  //Constant upload file
+
+    /**
+     * convertImage method receives a image to covert.
+     * @param file contains the image file.
+     * @param inputPathFile contains the input path of the image.
+     * @param outputPathFile contains the output path of image converted.
+     * @param outputPathThumbnail contains the output path of thumbnail converted.
+     * @param widthOfFile number of image width.
+     * @param heightOfFile number of image height.
+     * @param whiteBlankPercentage percentage of whiteBlanck.
+     * @param degreesToRotate degrees of rotate.
+     * @return get the path of the upload file.
+     */
+    @PostMapping()
+    public String imageConvert(
+            @RequestParam("file") MultipartFile file,
+            @RequestParam String inputPathFile,
+            @RequestParam String outputPathFile,
+            @RequestParam String outputPathThumbnail,
+            @RequestParam String widthOfFile,
+            @RequestParam String heightOfFile,
+            @RequestParam String whiteBlankPercentage,
+            @RequestParam String degreesToRotate) {
+
+        try {
+            byte[] bytes = file.getBytes();
+            Path path = Paths.get(UPLOADED_FOLDER + file.getOriginalFilename());
+            Files.write(path, bytes);
+        } catch (IOException e) {
```

Comment:
the exception type is it correct in this case?
can you check if that type of exception is the correct?

Reply:
In javacodegeeks.com says:
IOException has sub classes such as FileNotFoundException, EOFException, UnsupportedEncodingException, SocketException, and SSLException. If the file is not found, FileNotFoundException is thrown. While reading a file, EOFException occurs when end of the file is reached. If the file has an unsupported encoding, UnsupportedEncodingException occurs.

Reply:
ok, if it is the correct one it os ok.